test_scripts/Polices/External_UCS/External_Consent_OFF/011_ATF_Policies_External_Consent_OFF_through_ignition_cycles.lua

```diff
@@ -0,0 +1,274 @@
+--------------------------------------Requirement summary---------------------------------------------
+--[Policies] External UCS: "OFF" status between ignition cycles
+
+------------------------------------General Settings for Configuration--------------------------------
+require('user_modules/all_common_modules')
+local common_functions_external_consent = require('user_modules/shared_testcases_custom/ATF_Policies_External_Consent_common_functions')
+local common_steps = require('user_modules/common_steps')
+local common_functions = require ('user_modules/common_functions')
+
+---------------------------------------Common Variables-----------------------------------------------
+--local policy_file = config.pathToSDL .. "storage/policy.sqlite"
+
+---------------------------------------Preconditions--------------------------------------------------
+-- Start SDL and register application
+common_functions_external_consent:PreconditonSteps("mobileConnection","mobileSession")
+-- Activate application
+common_steps:ActivateApplication("Activate_Application_1", config.application1.registerAppInterfaceParams.appName)
+
+-------------------------------------------Functions--------------------------------------------------
+local function CheckGroup001IsNotConsentedAndGroup002IsConsented()
+  --------------------------------------------------------------------------
+  -- Main check:
+  -- RPC of Group001 is disallowed to process.
+  --------------------------------------------------------------------------
+  Test["TEST_NAME_OFF" .. "_MainCheck_RPC_of_Group001_is_disallowed"] = function(self)
+    --mobile side: send SubscribeWayPoints request
+    self.mobileSession:SendRPC("SubscribeWayPoints",{})
+    --mobile side: SubscribeWayPoints response
+    EXPECT_RESPONSE("SubscribeWayPoints", {success = false , resultCode = "USER_DISALLOWED"})
+    EXPECT_NOTIFICATION("OnHashChange")
+    :Times(0)
+  end
+
+  --------------------------------------------------------------------------
+  -- Main check:
+  -- RPC of Group002 is allowed to process.
+  --------------------------------------------------------------------------
+  Test["TEST_NAME_OFF" .. "_MainCheck_RPC_of_Group002_is_allowed"] = function(self)
+    self.mobileSession:SendRPC("SubscribeVehicleData",{rpm = true})
+    EXPECT_HMICALL("VehicleInfo.SubscribeVehicleData")
+    :Do(function(_,data)
+        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS",{})
+      end)
+    EXPECT_RESPONSE("SubscribeVehicleData", {success = true , resultCode = "SUCCESS"})
```

Comment:
@istoimenova, Please update with usage of _correlationId_ because current version can be unstable